tests/core_tests/test_context.py

```diff
@@ -1098,16 +1100,47 @@ class TestSerialize(TestContext):
     def setUp(self):
         super(TestSerialize, self).setUp()
         # params used in creating contexts
+        # Add data to mocked shotgun
+        self.task = self.mockgun.create(
+            "Task",
+            {
+                "content": "task_content",
+                "project": self.project,
+                "entity": self.shot,
+                "step": self.step
+            }
+        )
+
+        self.version = self.mockgun.create(
+            "Version",
+            {
+                "code": "version_code",
+                "project": self.project
+            }
+        )
+
+        self.user = self.mockgun.create(
+            "HumanUser",
+            {
+                "name": "user_name"
+            }
+        )
+
         self.kws = {}
         self.kws["tk"] = self.tk
         self.kws["project"] = self.project
         self.kws["entity"] = self.shot
         self.kws["step"] = self.step
-        self.kws["task"] = {"id": 45, "type": "Task"}
```

Comment:
Bad test data. Using non-existing entities from Mockgun. More on why this change was introduced bellow.